src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/_params.py

```diff
@@ -30,6 +30,8 @@ def load_arguments(self, _):
         c.argument('location', get_location_type(self.cli_ctx),
                    help='Location of your IoT Central application. Default is the location of target resource group.')
         c.argument('sku', arg_type=get_enum_type(AppSku),
-                   help='Pricing tier for IoT Central applications. Default value is F1, which is free.')
+                   help='Pricing tier for IoT Central applications. Default value is S1.')
         c.argument(
             'subdomain', help='Subdomain for the IoT Central URL. Each application must have a unique subdomain.')
+        c.argument('template', help='IoT Central application template name. Default is a blank application.')
```

Comment:
Not sure if we want to change `blank application` with `custom application` to follow the UI

Reply:
that makes sense, i can change